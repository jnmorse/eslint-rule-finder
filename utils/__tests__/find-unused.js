const path = require('path')
const LoadConfig = require('../load-config')
const findUnused = require('../find-unused')

const configFile = path.resolve(__dirname, '../__fixtures__/eslintrc.json')

let config

beforeEach(() => {
  config = new LoadConfig(configFile)
})

test('finds unused rules', () => {
  const unusedRules = findUnused(config)
  expect(unusedRules.size).toMatchSnapshot()
})

test('find unused rules that are deprecated', () => {
  const unusedRules = findUnused(config, true)
  expect(unusedRules.size).toMatchSnapshot()
})
