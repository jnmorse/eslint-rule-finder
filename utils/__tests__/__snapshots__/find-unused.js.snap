// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`find unused rules that are deprecated 1`] = `251`;

exports[`finds unused rules 1`] = `243`;
